SimplenoteUITests/SimplenoteUISmokeTestsSettings.swift

```diff
@@ -0,0 +1,56 @@
+import XCTest
+
+class SimplenoteUISmokeTestsSettings: XCTestCase {
+
+    override class func setUp() {
+        app.launch()
+        getToAllNotes()
+        _ = attemptLogOut()
+        EmailLogin.open()
+        EmailLogin.logIn(email: testDataExistingEmail, password: testDataExistingPassword)
+        NoteList.waitForLoad()
+    }
+
+    override func setUpWithError() throws {
+        getToAllNotes()
+        NoteList.trashAllNotes()
+        Trash.empty()
+        NoteList.openAllNotes()
+    }
+
+    override class func tearDown() {
+        Settings.open()
+        Settings.condensedModeDisable()
+        Settings.close()
+    }
+
+    func testUsingCondensedNoteList() throws {
+        trackTest()
+        let note = NoteData(
+            name: "Condensed Mode Test",
+            content: "Condensed Mode Content",
+            tags: []
+        )
+
+        trackStep()
+        NoteList.openAllNotes()
+        NoteList.createNoteThenLeaveEditor(note)
+        NoteListAssert.noteContentIsShown(note)
+
+        trackStep()
+        Settings.open()
+        Settings.condensedModeEnable()
+        Settings.close()
+        NoteList.openAllNotes()
+        NoteListAssert.noteContentIsShownInSearch(noteName: note.name, expectedContent: "")
+        NoteListAssert.noteHeight(note, 44.0)
+
+        trackStep()
+        Settings.open()
+        Settings.condensedModeDisable()
+        Settings.close()
+        NoteList.openAllNotes()
+        NoteListAssert.noteContentIsShown(note)
+        NoteListAssert.noteHeight(note, 81.0)
```

Comment:
For now using the exact cell height as one of the ways to assert the condensed mode. If it proves to be flaky, it looks like we can add an accuracy parameter.

Reply:
I like the accuracy parameter idea. Let's hope we don't need to use it though 😄 .

One suggestion I have is to [_Replace Magic Number_](https://refactoring.com/catalog/replaceMagicLiteral.html). The `44.0` and `81.0` are clearly expected notes height, but where do they come from? It would be good to extract them in constants to make that clearer. 

```swift
let defaultIOSCellHeight = 44.0
let customExpandedModeCellHeight = 81.0
```

I'm guessing those5 are the meaning of those values, but I'm not sure.

Reply:
The funny part is, I initially had them as constant variables `cellHeightUsual` and `cellHeightCondensed`, but then removed them, thinking that you will propose to remove them, since they are used only once 😆 ... I'll revert to using them 🙂